### PR TITLE
Upgrade peer dependencies.

### PR DESCRIPTION
**Pull request by JSAssassin**
Addresses #3. This upgrades bedrock-permission-http peerDependencies to the latest.

### CODE REVIEW

**Comment by aljones15** [comment]
@JSAssassin has this been tested with any other projects?

File: package.json [hunk 1]
```diff
@@ -22,7 +22,7 @@
     "bedrock-express": "^2.0.3",
     "bedrock-mongodb": "^3.0.0",
```
Comment (aljones15) [inline package.json:23]: so sorry bedrock-mongodb obviously needs an update too! I am so sorry @JSAssassin

**Review by aljones15 — APPROVED** [review]
This looks good. THANKS!

**Review by mattcollier — APPROVED** [review]
Thanks!

**Pull request merged**